Xamarin.Forms.Core/Items/ItemsLayoutTypeConverter.cs

```diff
@@ -7,15 +8,46 @@ public class ItemsLayoutTypeConverter : TypeConverter
 	{
 		public override object ConvertFromInvariantString(string value)
 		{
-			if (value == "HorizontalList")
+			if (value == null)
```

Comment:
null or should we check string.IsNullOrWhiteSpace? or string.IsNullOrEmpty?

Reply:
On empty string InvalidOperationException with text "Cannot convert "" into Xamarin.Forms.IItemsLayout" is thrown. I think it is more descriptive than "Value cannot be null."

Reply:
I'm not sure if we mean the same thing. I was suggesting, shouldn't we change it to: `if (string.IsNullOrWhitespace(value))`